Updated PPTX functions and added flex dashboard template

collinschwantes opened the pull request:
Addressed issues 11:13 

Flex dashboard template is not entirely derived from function like `eha_pptx` because the theme elements were being incorporated properly. Passing them as additional arguments via YAML in the skeleton.rmd resolved the issue.

emmamendelsohn commented:
This is great! Played with the template and works as expected. Everything is nicely commented and clear. Just a couple notes:

Fix typo that is breaking package install: https://github.com/collinschwantes/ehastyle/blob/master/R/eha_flexdashboard.R#L11

Was wondering if we should have a version of the template that avoids `opencage`? The API key increases the barrier to entry. Since you are using a predefined list of cities, perhaps you could hardcode the lat/long values and comment out the `oc_forward()` function (so that folks can still see how it's done).

collinschwantes commented:
@emmamendelsohn I fixed the typo and added a locations dataset to eliminate the `opencage` dependency.  Do we have a preferred geocoding package/platform?

emmamendelsohn commented:
I don't think so. We've used the Google API, but `opencage` works well too. It's more about making it slightly easier for folks to run the template in case they're not set up one way or another.

collinschwantes commented:
I totally agree with getting rid of the geocode components from the
template to make it easier to use.

Good to know that we’ve used Google in the past.


On Thu, Jun 24, 2021 at 6:23 PM Emma Mendelsohn ***@***.***>
wrote:

> I don't think so. We've used the Google API, but opencage works well too.
> It's more about making it slightly easier for folks to run the template in
> case they're not set up one way or another.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ecohealthalliance/ehastyle/pull/16#issuecomment-867992001>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOTF3ICTO674KORHYUNLPLTUOV4LANCNFSM464OQ4ZA>
> .
>

Disclaimer

The information contained in this communication from the sender is confidential. It is intended solely for use by the recipient and others authorized to receive it. If you are not the recipient, you are hereby notified that any disclosure, copying, distribution or taking action in relation of the contents of this information is strictly prohibited and may be unlawful.

This email has been scanned for viruses and malware, and may have been automatically archived by Mimecast, a leader in email security and cyber resilience. Mimecast integrates email defenses with brand protection, security awareness training, web security, compliance and other essential capabilities. Mimecast helps protect large and small organizations from malicious activity, human error and technology failure; and to lead the movement toward building a more resilient world. To find out more, visit our website.

collinschwantes commented:
addresses issues: #11  #12 #13
